src/renderer/App.vue

```diff
@@ -78,7 +79,8 @@ body,
     background-color: #d9e1e8;
   }
 
-  --specified-fonts: 'Noto Sans', 'Noto Sans CJK JP', 'Takaoゴシック', 'ヒラギノ角ゴ ProN W3', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Helvetica Neue', 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
+  --specified-fonts: 'Noto Sans', 'Noto Sans CJK JP', 'Takaoゴシック', 'ヒラギノ角ゴ ProN W3', '-apple-system', 'BlinkMacSystemFont',
```

Comment:
__[stylelint]__ Expected newline after "," in a multi-line list (value-list-comma-newline-after) ([link](https://sider.review/gh/repos/124234495/pulls/935#issue-4174478))

<sub>posted by Sider</sub>